src/actions/plugins.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Client4} from 'client';
+import {bindClientFunc} from './helpers';
+
+import {PluginTypes} from 'action_types';
+
+export function getMarketplacePlugins() {
+    return bindClientFunc({
+        clientFunc: Client4.getMarketplacePlugins,
+        onSuccess: PluginTypes.RECEIVED_MARKETPLACE_PLUGINS,
+    });
```

Comment:
@marianunez do you know what happens on the failure case? does it not do anything we have to handle that explicitly?

Reply:
Yes, there is a onFailure case that will handle in a commit soon. WIP